apps/server/src/modules/me/api/mapper/me.response.mapper.ts

```diff
@@ -0,0 +1,124 @@
+import { School } from '@src/modules/school';
+import { Role, User } from '@shared/domain/entity';
+import { EntityId } from '@shared/domain/types';
+import {
+	MeAccountResponse,
+	MeResponse,
+	MeRolesReponse,
+	MeSchoolLogoResponse,
+	MeSchoolResponse,
+	MeUserResponse,
+} from '../dto';
+
+export class MeResponseMapper {
+	public static mapToResponse(school: School, user: User, accountId: EntityId): MeResponse {
+		const schoolResponse = MeResponseMapper.mapSchool(school);
+		const userResponse = MeResponseMapper.mapUser(user, school);
+		const rolesResponse = MeResponseMapper.mapUserRoles(user);
+		const permissionsResponse = MeResponseMapper.mapPermissions(user);
+		const accountResponse = MeResponseMapper.mapAccount(accountId);
+
+		const res = new MeResponse({
+			school: schoolResponse,
+			user: userResponse,
+			roles: rolesResponse,
+			permissions: permissionsResponse,
+			account: accountResponse,
+		});
+
+		return res;
+	}
+
+	private static mapSchool(school: School): MeSchoolResponse {
+		const schoolInfoProps = school.getInfo();
+		// i missed the school logoName and url ..is used in frontend but unkown on which place but should expose over me
```

Comment:
Was meinst du hier mit "i missed..."?

Reply:
Es ist irgendwo in Verwendung, da es in der Oberfläche richtig angezeigt wird. Aber es scheint durch eine seperate Aktion, oder das seperate get school weiter gereicht zu werden. Daher muss dort aufjedenfall die Verdratung in Vue erfolgen.
Aber ich habe die Codezeile nicht direkt gefunden.